### ISSUE TITLE
Consider moving Fidget to its own repo?

### ISSUE THREAD

**Issue by ross-spencer** [openpreserve/format-corpus#20]
There are log4j dependencies in Fidget, dependabot seems to be returning [these files](https://github.com/openpreserve/format-corpus/search?q=log4j).

Related to https://github.com/openpreserve/format-corpus/issues/9 perhaps migrating it into its own repository would be a logical start, then the format-corpus would be more independently a standalone format corpus? 

Whether it is moved or not, there's a wiki page, but most of the information there looks like it would fit into the fidget README.md which seems like the first place folks might look for that? (If it is moved then those pages would need fixing up)

* https://github.com/openpreserve/format-corpus/wiki/Using-Fidget
* Also linked to from: https://github.com/openpreserve/format-corpus/wiki

**Comment by anjackson** [comment]
Yeah, I mean, it's log4j 1 not 2, but that's not great either.  Perhaps the whole tools section should just be deleted? Is anyone using any of it?  I'm unlikely to have much time to look after the parts I wrote, and if no-one else is using it then maybe it's best to let it go.

I'll ping @bitsgalore as I see some commits from him in the tools area, in case he's using them.

**Comment by anjackson** [comment]
@ross-spencer Yes, I think that was the general idea, a long time ago, before I had heard of/understood TrID, I think!

I prepped a PR to move this into a separate repo, and will prep a PR to drop it from here.

**Comment by anjackson** [comment]
Note that the wiki page(s) referencing the tools should be moved over too. Probably as plain files or just in the README.
